raiden-cli/src/routes/channels.ts

```diff
@@ -74,9 +170,13 @@ export function makeChannelsRouter(this: Cli): Router {
 
   router.put('/', openChannel.bind(this));
 
-  router.patch('/:tokenAddress/:partnerAddress', (_request: Request, response: Response) => {
-    response.status(404).send('Not implemented yet');
-  });
+  router.patch(
+    '/:tokenAddress/:partnerAddress',
+    validateOptionalAddressParameter.bind('tokenAddress'),
+    validateOptionalAddressParameter.bind('partnerAddress'),
```

Comment:
I think here the parameters are not optional. How to get the exact channel else?

Reply:
Ops, brain fart. Will fix.